### PR TITLE
[Launchpad Discovery]: Token discovery on explore page

### PR DESCRIPTION
**Pull request by mzparacha**
## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/8803

## Description of Changes
- Add token discovery list on explore page
- Token pricing (market cap, market cap goal, current price, 24hr price change) are static atm and will be added in a followup ticket https://github.com/hicommonwealth/commonwealth/issues/9694

## "How We Fixed It"
N/A

## Test Plan
1. Enable `FLAG_TOKENIZED_COMMUNITY`
2. Visit `/communities`
3. Verify you see tokens list, if you have previously created any tokens, if not, verify you see an empty tokens placeholder, with an option to launch your own token.
4. Verify you can click on "see more" button to get more tokens (if total tokens in your local env is > 8)
5. Verify you see correct loading state per initial/subsequent loading of tokens
6. Verify that clicking on the token card redirects to its community
7. Verify cards are responsive across screen sizes
8. Verify you don't see the tokens list when `FLAG_TOKENIZED_COMMUNITY` is disabled

## Deployment Plan
N/A

## Other Considerations
N/A

### CODE REVIEW

File: packages/commonwealth/client/scripts/views/pages/Communities/Communities.tsx [hunk 1]
```diff
@@ -200,6 +209,8 @@ const CommunitiesPage = () => {
           </div>
           <IdeaLaunchpad />
         </div>
+        <TokensList />
```
Comment (masvelio) [inline packages/commonwealth/client/scripts/views/pages/Communities/Communities.tsx:212]: should we put it behind the featureflag?

File: packages/commonwealth/client/scripts/views/pages/Communities/TokensList/TokensList.tsx [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    hasNextPage,
+    fetchNextPage,
+  } = useFetchTokensQuery({
+    cursor: 1,
+    limit: 8,
+    enabled: tokenizedCommunityEnabled,
+  });
+  const tokens = (tokensList?.pages || []).flatMap((page) => page.results);
+
+  const handleFetchMoreTokens = () => {
```
Comment (masvelio) [inline packages/commonwealth/client/scripts/views/pages/Communities/TokensList/TokensList.tsx:30]: wondering how pagination will work for two lists in the same view (tokens  + communities)

**Review by masvelio — APPROVED** [review]
1. On mobile, scrollable page is not visible at all because of the banners/filters etc

https://github.com/user-attachments/assets/0d75f8d4-e4e8-4219-a1d0-7744544baefa

2. Should cards have different widths? 

image

3. Should we have some placeholder image if token does not have image?

4. Maybe we should use tabs for communities and tokens instead of putting them all in one page? Eg filters now are visible but works only for communities. Only my though for design team to consider

File: packages/commonwealth/client/scripts/views/pages/Communities/Communities.tsx (hunk 1 above)
Comment (mzparacha) [reply packages/commonwealth/client/scripts/views/pages/Communities/Communities.tsx:212]: that check is inside this component

https://github.com/hicommonwealth/commonwealth/blob/d4582ba003e772ea297b0ce636af4d40001efc74/packages/commonwealth/client/scripts/views/pages/Communities/TokensList/TokensList.tsx#L36

File: packages/commonwealth/client/scripts/views/pages/Communities/TokensList/TokensList.tsx (hunk 2 above)
Comment (mzparacha) [reply packages/commonwealth/client/scripts/views/pages/Communities/TokensList/TokensList.tsx:30]: The communities pagination works on scroll (like before), while this one involves user to click "load more" button.

**Comment by mzparacha** [comment]
1. In designs there are more assets/sections on this page and that will be even harder to navigate. I remember @Israellund raised this in a meeting (maybe design crit), I don't recall if we were going to update designs for this page or add separate pages for tokens/communities tho. But I agree that this is bloated ATM. 
2. Nope, fixed now. Ty for spotting.
3. Made it mandatory per confirmation from product.
4. Same as 1, we need some design changes for this to work on 1 page (or make separate page). 

cc: @masvelio

**Pull request merged**